src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -502,21 +504,19 @@ impl EventProcessor {
             }
 
             self.dnd.source_window = Some(source_window);
-            if self.dnd.result.is_none() {
```

Comment:
why this got removed?

Reply:
`self.dnd.result.is_none()` is only `true` for the first `DragEnter` event. Keeping this check means no `DragOver` (aka `DragMoved`) events will be fired.